src/graph.rs

```diff
@@ -20,18 +24,16 @@ pub(crate) struct GraphData {
     stopping: Option<DateTime<Utc>>,
     /// Tracks when the load test stopped with an optional system timestamp.
     stopped: Option<DateTime<Utc>>,
-    /// Counts requests per second. Each element of the vector represents one second.
-    requests_per_second: Vec<u32>,
-    /// Counts errors per second. Each element of the vector represents one second.
-    errors_per_second: Vec<u32>,
-    /// Maintains average response time per second. Each element of the vector represents one second.
-    average_response_time_per_second: Vec<MovingAverage>,
-    /// Number of tasks at the end of each second of the test. Each element of the vector
-    /// represents one second.
-    tasks_per_second: Vec<usize>,
-    /// Number of users at the end of each second of the test. Each element of the vector
-    /// represents one second.
-    users_per_second: Vec<usize>,
+    /// Counts requests per second for each request type.
+    requests_per_second: HashMap<String, TimeSeries<u32, u32>>,
```

Comment:
(Followup is fine) it seems worthwhile to create a custom Type for `HashMap<String, TimeSeries<u32, u32>>`, something like `ItemsPerSecond` or `PerSecondCounter` or ...? I see it throughout the new code, and we'd document it thoroughly where the type is created.